androidbrowserhelper/build.gradle

```diff
@@ -18,11 +18,11 @@ apply plugin: 'com.android.library'
 apply plugin: 'maven-publish'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 30
```

Comment:
How stable is this?

Reply:
This will only affect devices that have Android 11. Otherwise, doesn't have any side-effects.